frontend/catalyst/debug/compiler_functions.py

```diff
@@ -335,6 +335,7 @@ def f(x):
         f"-Wl,-rpath,{python_lib_dir_path}",
         f"-L{python_lib_dir_path}",
         "-lpython" + version_str,
+        "-lcurses",
```

Comment:
I got this error after patching LLVM.

```shell
llvm-project/build/lib/libmlir_async_runtime.so.19.0git: undefined reference to `setupterm@NCURSES6_TINFO_5.0.19991023'
```

Reply:
It looks like Support adds adds these:

llvm/lib/Support/Unix/Process.inc

```cxx
#ifdef LLVM_ENABLE_TERMINFO
// We manually declare these extern functions because finding the correct
// headers from various terminfo, curses, or other sources is harder than
// writing their specs down.
extern "C" int setupterm(char *term, int filedes, int *errret);
extern "C" struct term *set_curterm(struct term *termp);
extern "C" int del_curterm(struct term *termp);
extern "C" int tigetnum(char *capname);
#endif
```

But not sure how they got in yours... :thinking:

Reply:
I rerun the wheel without lcurses and it seems to work.  Maybe I messes up my local environment and previous wheel attempts. The library is removed right now.